src/structures/session.ts

```diff
@@ -754,6 +795,19 @@ export default abstract class Session {
 
             this.stopGuessTimeout();
 
+            if (this.isGameSession() && this.isClipMode()) {
```

Comment:
clipmode => game session?

Reply:
I needed this to prevent casting `session` to a game session, as its currently scoped as `Session` here